Makefile

```diff
@@ -138,7 +109,6 @@ KUBE_API := $(or ${KUBE_API},no)
 ifeq ($(ENABLE_KUBE_API),true)
```

Comment:
I don't understand how ``ENABLE_KUBE_API`` here might be ``true`` and downwards can be ``no``

Reply:
I think it's relevant just for this `if` block

Reply:
Not sure I understand what you're saying
So we can change it to yes/no values everywhere?

Reply:
Not sure if that you meant but this is how it was configured on most of test-infra boolean variables.
We need to find a better way for setting boolean value